### ISSUE TITLE
Finish porting gl-matrix modules over

### ISSUE THREAD

**Issue by mikolalysenko** [stackgl/contributing#22]
Still need:
- gl-quaternion
- gl-vec2
- gl-vec3

Also nice to have:
- Plucker coordinates for lines
- Conic sections?
- Maybe some module for working with exterior forms?

**Comment by marklundin** [comment]
Is this still valid? If so I could take a look.

**Comment by mikolalysenko** [comment]
Yep.  If you want to take it on, go for it!

**Comment by backspaces** [comment]
Note: Toji has refactored his library for use with webpack. I _think_ this could make it much easier for us to keep up with his updates.  It uses require() so is npm ready, I think.

My interest is that I'm accessing all these goodies via jspm which seem pretty agile.  I can likely get the source files w/ requires and jspm will grok them.

```
https://github.com/toji/gl-matrix/tree/master/src
```

.. rather than the 'dist" files of the entire package, with webpack annotations:

```
https://github.com/toji/gl-matrix/tree/master/dist
```

Basically, its kinda hard for us to follow Toji's releases, so we may be better off using his webpack modularization as a starting point.

**Comment by mattdesl** [comment]
bit more discussion on that [here](https://github.com/stackgl/gl-mat3/issues/1#issuecomment-113783244)
